fbw-common/src/wasm/systems/systems/src/navigation/adirs.rs

```diff
@@ -3201,6 +3333,110 @@ mod tests {
             );
         }
 
+        #[rstest]
+        #[case(1)]
+        #[case(2)]
+        #[case(3)]
+        fn max_airspeed_is_provided_at_sea_level(#[case] adiru_number: usize) {
+            let mut test_bed = all_adirus_aligned_test_bed();
+            test_bed.set_ambient_pressure(Pressure::new::<hectopascal>(1013.25));
+            test_bed.run();
+
+            assert_true!(test_bed.max_airspeed(adiru_number).value().get::<knot>() > 330.);
+        }
+
+        #[rstest]
+        #[case(1)]
+        #[case(2)]
+        #[case(3)]
+        fn max_airspeed_is_limited_by_mmo(#[case] adiru_number: usize) {
+            use ntest::assert_true;
+
+            let mut test_bed = all_adirus_aligned_test_bed();
+            // FL390
+            test_bed.set_ambient_pressure(Pressure::new::<hectopascal>(196.773));
+            test_bed.run();
+
+            assert_true!(test_bed.max_airspeed(adiru_number).value().get::<knot>() < 300.);
+        }
+
+        #[rstest]
+        #[case(1)]
+        #[case(2)]
+        #[case(3)]
+        fn overspeed_warning_is_inactive_at_low_speed_at_sea_level(#[case] adiru_number: usize) {
```

Comment:
Why is this test called "..._at_low_speed_..."? 300 knots is not really "low speed".

Reply:
Low speed below vmo/mmo, high speed above 😝 . I could change it to "normal" speed.